templates/app.scaleft.yaml

```diff
@@ -0,0 +1,18 @@
+identities:
+- node:
+    name: app_scaleft
+    dns_pattern: app.scaleft.com
+    dst_port: 443
+  pushdown: []
+- node:
+    name: sbin_sftd
+    zone: .*
+    app: .*
+    binary_name: /usr/sbin/sftd
```

Comment:
this should be an agent template. /usr/sbin/sftd is the agent. Why are you calling it app?

Reply:
fixed all these comments in my newest commit